src/NexusMods.App.UI/Controls/MarkdownRenderer/MarkdownRendererView.axaml.cs

```diff
@@ -12,12 +14,14 @@ public MarkdownRendererView()
 
         this.WhenActivated(disposables =>
         {
-            this.OneWayBind(ViewModel, vm => vm.Contents, view => view.MarkdownScrollViewer.Markdown)
-                .DisposeWith(disposables);
+            var viewModel = ViewModel!;
+
+            MarkdownScrollViewer.Engine.HyperlinkCommand = viewModel.OpenLinkCommand;
+            MarkdownScrollViewer.Engine.Plugins.PathResolver = viewModel.PathResolver;
+            MarkdownScrollViewer.Engine.Plugins.Plugins.Add(viewModel.ImageResolverPlugin);
```

Comment:
Should this be in `this.WhenAnyValue(v=>v.ViewModel).WhereNotNull().Do(PopulateFromVM())` or is out of that by design?

Reply:
I had before. Working with the markdown engine directly is already hacky enough.

Reply:
I can re-add it, shouldn't make a difference... hopefully

Reply:
You know about this better than me so up to you

Reply:
I'll keep this as is. It doesn't really make sense to react to VM changes in this context. I'd have to reset the entire markdown rendering engine and then add the new stuff.